scripts/lib/libvirt/admin-config

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+import argparse
+
+import libvirt_setup
+
+
+def main():
+
+    parser = argparse.ArgumentParser(description="Create Admin Node Config.")
+    parser.add_argument("cloud", type=str, help="Name of the cloud")
+    parser.add_argument("adminnodememory", type=str, help="Amount of Memory")
+    parser.add_argument("adminvcpus", type=str, help="Number of VCPUs")
+    parser.add_argument("emulator", type=str, help="Emulator")
```

Comment:
Which emulator?  I prefer to give an example here otherwise the help text is not very helpful.